python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -450,6 +450,12 @@ class GbdesAstrometricFitConfig(
         "returned are not used.",
         default="phot_g_mean",
     )
+    setRefEpoch = pexConfig.Field(
+        dtype=float,
+        doc="Set reference epoch (set to median if None). (units of input in mjd)",
```

Comment:
I'm trying to think of how to make description clear, how about: "Set the reference epoch to a fixed value in MJD (if None, median observation date is used)" ?